### ISSUE TITLE
data storage for Grappa

### ISSUE THREAD

**Issue by aaalgo** [uwsampa/grappa#241]
I cannot seem to find any documentation or discussion regarding the storage or filesystem to be used with Grappa.  I've looked a little bit into the code and I think HDFS is currently not supported yet.  Assume we have about 10 machines, and don't want data loading/writing to dominate the processing time, what kind of data storage should we use with Grappa?

Thank you very much!

**Comment by nelsonje** [comment]
We generally run on systems with a shared NFSv4, Lustre, or IBM GPFS filesystem. We have also mounted HDFS over FUSE in the past, but since we stopped using that on our cluster we've allowed some of our parallel write routines to assume that POSIX record locking is available. 

For small clusters, NFSv4 with a high-bandwidth network and a bunch of disks in some sort of RAID configuration works great. Our 24TB 12-disk NFS server with 40Gb InfiniBand has no problem reading a big file in parallel at 1GB/s.
